### DISCUSSION TITLE
How to draw shapes with a future timestamp in the Tradingview api?

### DISCUSSION THREAD

**Discussion by sangdx** [tradingview/lightweight-charts#1586] [Q&A]
I want to draw shape with `IChartWidgetApi`, first vertical line with timestamp now and second verify line but both of them is same position.
#first vertical line
chart().createShape(
      { time: new Date().getTime() / 1000 },
      {shape: 'vertical_line'}
);
#second vertical line
chart().createShape(
      { time: new Date().getTime() / 1000 + 60 },
      {shape: 'vertical_line'}
);
Is there any other way to create a vertical line that can achieve my wishes, or if I do the above, how can I fix it?

**Comment by elliot-derivio** [comment]
@sangdx Same issue here 🤔 Have you found a solution yet?
